src/components/dialog-parameters/make-slider.js

```diff
@@ -0,0 +1,43 @@
+import { Box, Grid, Slider, Typography } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import React, { useState } from 'react';
+import { useStyles } from './parameters';
+
+export function MakeSlider(
```

Comment:
should be better as 
const LabelledSilder = ({threshold, label, disabled, onCommitCallback thresholdMarks} ) =>

Reply:
it's corrected